### PR TITLE
Page responses should have an html tag

### PR DESCRIPTION
**Pull request by jpaddison3**
Fixes: https://github.com/ForumMagnum/ForumMagnum/issues/4945

### CODE REVIEW

File: packages/lesswrong/components/tagging/TagEditsByUser.tsx [hunk 1]
```diff
@@ -47,8 +47,7 @@ const TagEditsByUser = ({userId, limit, classes}: {
   }
 
   return <div className={classes.root}>
-    {resultsWithLiveTags.map(tagUpdates =>
-      <Components.SingleLineTagUpdates
+    {resultsWithLiveTags.map(tagUpdates => <Components.SingleLineTagUpdates
```
Comment (jpaddison3) [inline packages/lesswrong/components/tagging/TagEditsByUser.tsx:50]: I noticed an indentation error in my last PR right as I merged it.

File: packages/lesswrong/server/apolloServer.ts [hunk 2]
```diff
@@ -217,10 +218,11 @@ export function startWebserver() {
...
         + '</head>\n'
         + '<body class="'+classesForAbTestGroups(allAbTestGroups)+'">\n'
-          + ssrBody
-        +'</body>\n'
-        + embedAsGlobalVar("ssrRenderedAt", renderedAt)
-        + serializedApolloState)
+          + ssrBody + '\n'
+        + '</body>\n'
+        + embedAsGlobalVar("ssrRenderedAt", renderedAt) + '\n'
+        + serializedApolloState + '\n'
```
Comment (jpaddison3) [inline packages/lesswrong/server/apolloServer.ts:224]: All this is just newlines so I could read the page source a bit better.

File: packages/lesswrong/server/vulcan-lib/apollo-ssr/renderUtil.ts [hunk 3]
```diff
@@ -1,5 +1,3 @@
-import type { ThemeOptions } from '../../../themes/themeNames';
```
Comment (jpaddison3) [inline packages/lesswrong/server/vulcan-lib/apollo-ssr/renderUtil.ts:1]: Unused import

**Review by jimrandomh — APPROVED** [review]

**Comment by jimrandomh** [comment]
From a superficial reading of the MDN page, it sounds like `<html>` is optional, but it's where the `lang` attribute goes which I guess is helpful metadata for something, though I'm not aware of any problems having occurred as a result of it being missing. In any case, this certainly doesn't hurt anything.

**Review by Xodarap — APPROVED** [review]
thanks for the quick turnaround on this

**Pull request merged**